code/modules/projectiles/guns/smgs.dm

```diff
@@ -687,3 +687,10 @@
 	if(.)
 		click_empty(user)
 	return FALSE
+
+/obj/item/weapon/gun/smg/nailgun/compact/tactical
+	name = "tactical compact nailgun"
+	desc = "A carpentry tool, used to drive nails into tough surfaces. This one is military grade, it's olive drab and tacticool. Cannot fire nails as a projectile."
+	icon_state = "tnailgun"
+	item_state = "tnailgun"
+	w_class = SIZE_SMALL
```

Comment:
Also you may as well remove the combat stats on examine by setting `GUN_UNUSUAL_DESIGN` in its `flags_gun_features` if you're not going to allow this to be used in combat: 
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/61b77424-050b-407d-aef1-37740a0782a8)